examples/fluids/navierstokes.h

```diff
@@ -216,6 +216,11 @@ struct SetupContext_ {
   CeedScalar dc_axis[3];
   CeedScalar wind[3];
   CeedScalar time;
+  CeedScalar midpt;
```

Comment:
Nice work, @tt-aiken ! 
What is `midpt`? Is it `midpoint`?

Reply:
Thanks! 
Yes, this is midpt. Probably better to just use the full word for clarity.

Reply:
Yeah, `midpoint`, `mid_point`, or even `mid_pt` are easier to read, IMO.